pocs/mount/bisque.py

```diff
@@ -301,14 +302,14 @@ def write(self, value):
         return self.theskyx.write(value)
 
     def read(self, timeout=5):
+
         response_obj = {'success': False}
-        while True:
-            response = self.theskyx.read()
-            if response is not None or timeout == 0:
-                break
-            else:
-                time.sleep(1)
-                timeout -= 1
+
+        try:
+            response = self.theskyx.read(timeout=timeout)
+        except Exception as err:
+            self.logger.error(err)
```

Comment:
@wtgee, Not sure whether you'd rather the error be raised here.

Reply:
I think this was put in mostly as a debugging check because we were getting exceptions but didn't know why but wanted to.  Could be removed now.

Reply:
O.K., I'll remove it.